frontend/packages/knative-plugin/src/plugin.tsx

```diff
@@ -37,6 +38,16 @@ import { TopologyConsumedExtensions, topologyPlugin } from './topology/topology-
 import * as eventSourceIcon from './imgs/event-source.svg';
 import * as channelIcon from './imgs/channel.svg';
 import { eventSourceProvider, kameletsProvider } from './catalog';
+import {
+  eventSourceBreadcrumbsProvider,
+  channelBreadcrumbsProvider,
+  brokerBreadcrumbsProvider,
+} from './hooks';
+import {
+  eventSourceModelsProviderForBreadcrumbs,
+  channelModelsProviderForBreadcrumbs,
+  brokerModelProviderForBreadcrumbs,
+} from './utils';
```

Comment:
move these utilities to there respective hooks file as this is specific for breadcrumbs